### PR TITLE
Add app-cadet-list

### PR DESCRIPTION
**Pull request by Tuengdin**
Just try to add new app.

### CODE REVIEW

File: app/app/(app-active-cadets)/active-cadets/layout.tsx [hunk 1]
```diff
@@ -0,0 +1,16 @@
+export const metadata = {
+  title: 'Next.js',
+  description: 'Generated by Next.js',
+}
+
+export default function RootLayout({
```
Comment (hazamashoken) [inline app/app/(app-active-cadets)/active-cadets/layout.tsx:6]: Add <app name> to the rootlayout name
 
Easy debugging when things are specific

e.g `ActiveCadetsRootLayout`

File: app/app/(app-active-cadets)/active-cadets/page.tsx [hunk 2]
```diff
@@ -0,0 +1,4 @@
+const ActiveCadets = () => {
```
Comment (hazamashoken) [inline app/app/(app-active-cadets)/active-cadets/page.tsx:1]: For page components name 

It's recommended to have word `Page`
For the Page component 
e.g. `ActiveCadetPage`

And Page component should be a normal
function

`export default function ... Page () ... `

**Review by hazamashoken — CHANGES_REQUESTED** [review]

File: app/app/(app-active-cadets)/active-cadets/page.tsx (hunk 2 above)
Comment (Lodimup) [reply app/app/(app-active-cadets)/active-cadets/page.tsx:1]: Let's create NORM.md

**Comment by Lodimup** [comment]
I converted the PR to draft.

File: app/app/(app-active-cadets)/active-cadets/page.tsx (hunk 2 above)
Comment (Lodimup) [reply app/app/(app-active-cadets)/active-cadets/page.tsx:1]: https://github.com/42-Bangkok/portal/pull/54

Comment (Tuengdin) [reply app/app/(app-active-cadets)/active-cadets/page.tsx]: changed to  ActiveCadetPage

**Comment by Lodimup** [comment]
to be merged the core feature should be completed na.

**Pull request closed without merging**